src/main/java/com/magicvet/service/ClientService.java

```diff
@@ -19,12 +20,13 @@ public Client registerNewClient() {
         return client;
     }
 
-    private static Client buildClient() {
+    public Client buildClient() {
         Client client = new Client();
-        client.setEmail(getValidInput("Email : ", EMAIL_PATTERN, "(e.g. example@example.com)."));
+        client.setEmail(getValidInput("Email: ", EMAIL_PATTERN, "(e.g. example@example.com)."));
         String nameErrorMessage = "Name must contain 3 or more Latin letters, hyphen (-) is allowed.";
```

Comment:
You've changed the NAME_PATTERN for at least 2 letters, so the nameErrorMessage should be probably updated also.

Reply:
Certainly! It’s great that you’re paying attention to the details. Since I’ve adjusted the `NAME_PATTERN` to allow for at least 2 letters, updating the `nameErrorMessage` to reflect this change is a good idea.